tc-discord-webhooks-server/src/main/java/com/github/playerforcehd/tcdiscordwebhooks/discord/DiscordWebHookProcessor.java

```diff
@@ -84,6 +86,16 @@ public boolean sendDiscordWebHook(String webHookURL, String discordWebHookPayloa
             httpPost.addHeader("Accept-Language", "en-US,en;q=0.5");
             httpPost.addHeader("Content-Type", "application/json");
             httpPost.setEntity(new StringEntity(discordWebHookPayload, HTTP_CHARSET));
+            String httpProxyHost = System.getProperty("http.proxyHost");
+            String httpProxyPort = System.getProperty("http.proxyPort");
+            if (httpProxyHost!=null && httpProxyPort.trim().length()>0 && httpProxyPort!=null) {
+                Integer port = Integer.valueOf(httpProxyPort);
+                HttpHost proxy = new HttpHost(httpProxyHost, port, "http");
+                RequestConfig.Builder reqconfigconbuilder = RequestConfig.custom();
+                reqconfigconbuilder = reqconfigconbuilder.setProxy(proxy);
+                RequestConfig config = reqconfigconbuilder.build();
```

Comment:
Please use camel case for the RequestConfig builder (`reqConfigBuilder `).
In addition, please chain the calls of the builders methods together. You might also completly ommit the builders variable at all if you like:
```java
RequestConfig config = RequestConfig.custom()
    .setProxy(proxy)
    .build();
```